src/store/CategoricalDisplayStore.js

```diff
@@ -52,8 +52,9 @@ export default class CategoricalDisplayStore {
         emptyDescription: t.translate('There are no favorites yet.'),
         data: [],
     };
+    @observable hideLookupFilter = false;
     scrollTop = undefined;
-
+    autoHideLookupFilter = true;
```

Comment:
why do we need this variable?

Reply:
@brucebinary When user search or moving through categories by selecting them, the scroll moves up and down as well, its inconvenient if we dispaear the seach text or category-select when user are using them. this is tell not to hide them on those cases.

Reply:
@MahboobehMohammadi I see. It helps that you find a way to make that intent more clearly communicated. Either better names or just a comment. Future developers reading your code should not need to do detective work on your code.

Reply:
@MahboobehMohammadi comments should be a last resort.

Reply:
@brucebinary how about `isUserScrolling` ? this pretty much explain it.

Reply:
That's a better name 👍 . A short comment on the scroll event being triggered when the user keys in something would be helpful as well.